src/login/commands/loginToCloud.ts

```diff
@@ -40,7 +40,7 @@ export async function loginToCloud(context: IActionContext): Promise<void> {
 			}
 			const tenantId: string | undefined = await window.showInputBox({
 				prompt: localize('azure-account.enterTenantId', "Enter the tenant id"),
-				placeHolder: localize('azure-account.tenantIdPlaceholder', "Enter your tenant id, or '{0}' for the default tenant", commonTenantId),
+				placeHolder: localize('azure-account.tenantIdPlaceholder', 'Enter your tenant id or leave it blank to use the default tenant'),
```

Comment:
Did blank already get assumed to mean common tenant ID elsewhere in the code?

Reply:
Yes we always default to the common tenant ID if that setting is undefined or blank. Here's a few examples:

https://github.com/microsoft/vscode-azure-account/blob/615ab01860abcc5ce76bb1caa490975d66a9efa3/src/login/AzureLoginHelper.ts#L138

https://github.com/microsoft/vscode-azure-account/blob/615ab01860abcc5ce76bb1caa490975d66a9efa3/src/login/AzureLoginHelper.ts#L190